suricata/types.json

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
It feels a bit rich to have a top-level directory just for this file. I did it to follow what we did for Zeek, but I'd be game to move this file to the https://github.com/brimsec/build-suricata repo if people want it. Preferably in a follow on PR.

Reply:
Actually (simpler to do) it could go in zqd/ingest alongside the go generated suricata.go.

Reply:
My vote would be to put this in `zqd/ingest`. *Or* we could have a top level folder called `jsontypes` and put the type configs for suricata and zeek json.

Reply:
*Or* we could do what we do with log ingest and pcap ingest a form post and allow the suricata type config to be a value in the form.

Reply:
i can put it in `zqd/ingest`. I'd rather not change the pcap API for now since (unlike for log ingest) this isn't currently intended to be user configurable.